Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.yml

```diff
@@ -2229,7 +2228,629 @@ script:
+      - contextPath: GSuite.User.includeInGlobalAddressList
+        description: Indicates if the user's profile is visible in the G Suite global address list when the contact sharing feature is enabled for the domain.
+        type: Boolean
+      - contextPath: GSuite.User.thumbnailPhotoUrl
+        description: Photo Url of the user.
+        type: String
+      - contextPath: GSuite.User.thumbnailPhotoEtag
+        description: ETag of the user's photo.
+        type: String
+      - contextPath: GSuite.User.customSchemas
+        description: Custom fields of the user.
+        type: Unknown
+  - arguments:
+      - description: "The customerId is a unique identifier of the customer's Google account."
+        name: customer_id
+        required: true
+      - description: 'List of unique device IDs (comma separated) of Chrome Browser Devices to move. A maximum of 600 browsers may be moved per request.'
+        name: resource_ids
+        required: true
+      - description: "Destination organization unit to move devices to. Full path of the organizational unit or its ID prefixed with 'id:'."
+        name: org_unit_path
+        required: true
+    description: To move Chrome Browser devices assigned to an account from one organization unit to another, use the following POST request and include the access token obtained by following the "Authorization" section.
+    name: gsuite-chromebrowserdevice-move-ou
+  - arguments:
+      - description: The customerId is a unique identifier of the customer's Google account.
+        name: customer_id
+        required: true
+      - description: The unique ID of the device.
+        name: device_id
+      - auto: PREDEFINED
+        description: Field according to which to order by.
+        name: order_by
+        predefined:
+          - id
+          - last_sync
+          - machine_name
+          - extension_count
+          - policy_count
+          - os_version
+          - last_signed_in_user
+          - annotated_user
+          - annotated_location
+          - annotated_asset_id
+          - notes
+          - browser_version_channel
+          - org_unit
+          - enrollment_date
+          - save_browsing_clickthrough
+          - platform_major_version
+          - last_activity
+          - browser_version_sortable
+          - os_version_sortable
+      - description: The full path of the organizational unit or its unique ID.
+        name: org_unit_path
+      - auto: PREDEFINED
+        description: Restrict information returned to a set of selected fields. "BASIC" - Includes only the basic metadata fields (i.e., the ones found in the columns of the admin console browser list as enumerated above). "FULL" - Includes all metadata fields (as enumerated above).
+        name: projection
+        predefined:
+          - BASIC
+          - FULL
+      - description: Search string using the list page query language described in the section below (Filter Query Language).
```

Comment:
There is no section called Filter Query Language in this doc. Perhaps you need to add a link here to an external doc?

Reply:
No, it was a bad copy from google documentation. fixed